ibm/provider/provider.go

```diff
@@ -707,7 +707,6 @@ func Provider() *schema.Provider {
 			"ibm_cd_toolchain_tool_keyprotect":         cdtoolchain.DataSourceIBMCdToolchainToolKeyprotect(),
 			"ibm_cd_toolchain_tool_secretsmanager":     cdtoolchain.DataSourceIBMCdToolchainToolSecretsmanager(),
 			"ibm_cd_toolchain_tool_bitbucketgit":       cdtoolchain.DataSourceIBMCdToolchainToolBitbucketgit(),
-			"ibm_cd_toolchain_tool_githubintegrated":   cdtoolchain.DataSourceIBMCdToolchainToolGithubintegrated(),
```

Comment:
Is the res/ds releated to ibm_cd_toolchain_tool_githubintegrated removed ?
Please follow the gudelines suggested by Hashicorp to
https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/deprecations